client/lib/media-serialization/strategies/dom.js

```diff
@@ -21,7 +21,7 @@ function parseImage( node, _parsed ) {
 	_parsed.type = MediaTypes.IMAGE;
 	_parsed.media.URL = node.getAttribute( 'src' );
 	_parsed.media.alt = node.getAttribute( 'alt' );
-	_parsed.media.transient = ( 0 === ( _parsed.media.URL || '' ).indexOf( 'blob:' ) );
+	_parsed.media.transient = !! node.getAttribute( 'data-istransient' );
```

Comment:
The behavior of many attributes is such that if an attribute exists and is not explicitly `"false"`, it is assumed to be `true`, even if empty.

Example: https://codepen.io/aduth/pen/NvjwoN

Since this is a custom attribute it doesn't matter so much. The alternative is a little more verbose too:

```js
_parsed.media.transient = node.hasAttribute( 'data-istransient' ) && 'false' !== node.getAttribute( 'data-istransient' );
```